templates/actors/custos-sheet.html

```diff
@@ -6,32 +6,39 @@
         <section class="header-details">
             {{!-- Custos Summary --}}
 			<ul class="flexrow">
-				<li><img class="profile" src="{{actor.img}}" title="{{actor.name}}" data-edit="img"/></li>
+				<li><div class="portrait">
+                    <img class="laurel" src="./systems/LexArcana/styles/images/laurel1.webp"/>
+                    <img class="profile" src="{{actor.img}}" title="{{actor.name}}" data-edit="img"/>
+                    <img class="laurel" src="./systems/LexArcana/styles/images/laurel2.webp"/>
+                </div></li>
 				<li>
 					<h1 class="charname">
 						<input name="name" type="text" value="{{actor.name}}" placeholder="{{ localize 'LexArcana.Name' }}"/>
 					</h1>
-				</li>
```

Comment:
It seems to me there's a mismatch between <ul> elements and <li> elements can you confirm they all are closed correctly ?

Reply:
no, it's ok I just added a new ul inside the first ul it's just bad indentation that makes it look unclosed, sorry my bad.